Library/Homebrew/.rubocop_todo.yml

```diff
@@ -36,3 +36,13 @@ Lint/EmptyBlock:
     - "test/formula_validation_spec.rb"
     - "test/pathname_spec.rb"
     - "test/support/fixtures/cask/Casks/*flight*.rb"
+
+Homebrew/MoveToExtendOS:
+  Exclude:
+    - "cask/cmd/install.rb"
+    - "cmd/update-report.rb"
+    - "default_prefix.rb"
+    - "dev-cmd/bottle.rb"
+    - "dev-cmd/tests.rb"
+    - "dev-cmd/update-test.rb"
+    - "formula_cellar_checks.rb"
```

Comment:
Sorry for all the back and forth: what do you think about individual `#rubocop:disable` comments for these explaining each one and adding an inline `TODO` to go clean/fix them up? I think that's probably more likely to end up fixed than in here.

Reply:
The other advantage is that it will trigger a linter error if people try to add more of those OS checks to those files in the future. I like the idea.